thonny/workbench.py

```diff
@@ -181,6 +181,20 @@ def __init__(self, parsed_args: Dict[str, Any]) -> None:
 
         tk.Tk.__init__(self, className="Thonny")
         tk.Tk.report_callback_exception = self._on_tk_exception  # type: ignore
+
```

Comment:
Could you please extract this into separate method?

Reply:
Done!